consensus/hotstuff/cruisectl/config.go

```diff
@@ -1,27 +1,23 @@
 package cruisectl
 
 import (
-	"math"
 	"time"
 )
 
 // DefaultConfig returns the default config for the BlockRateController.
 func DefaultConfig() *Config {
 	return &Config{
-		TargetTransition: EpochTransitionTime{
-			day:    time.Wednesday,
-			hour:   19,
-			minute: 0,
-		},
+		TargetTransition: DefaultEpochTransitionTime(),
 		// TODO confirm default values
 		DefaultProposalDelay: 500 * time.Millisecond,
 		MaxProposalDelay:     1000 * time.Millisecond,
```

Comment:
We need to be careful how we configure it, `MaxProposalDelay + block building time` must be less than block timeout, otherwise we will undermine liveness. Sanity check somewhere would be ideal.

Reply:
I'm a bit on the fence about this.

One the one hand, `MaxProposalDelay` is already in a sense intended to be the sanity check which constrains the natural behaviour of the controller. 

In addition, if the error is very high, then overshooting the timeout is the right behaviour in a sense. We have purposefully done this (with a subset of nodes) before for exactly this reason. This is a bit different, because by default all nodes will introduce roughly the same delay, and if all nodes are delaying past the timeout that's obviously bad. But we have discussed operating a subset of nodes with a higher degree of freedom to adjust proposal delays, in which case allowing values beyond the timeout could still be valuable.

Reply:
I just want to avoid a situation where super-majority is configured incorrectly because `MaxProposalDelay >  round timeout` which undermines liveness for whole network.

Reply:
I am not worried about timeouts for the following reasons:
* Eventually, the pacemaker will increase the timeout and blocks will be produced again.
* Having some views time out satisfies our goal of slowing down the network
* In practise, this will probably be rare, as the controller should prevent a scenario, where the accumulate a lot of error.

Reply:
@durkmurder also keep in mind that the notation is misleading here. `MaxProposalDelay` is _not_ the time by which we truly delay blocks. It is the max _duration_ of the view!